divisible.rb

```diff
@@ -0,0 +1,14 @@
+# Write a method that returns in an array all the numbers between 1 and 100 that are divisible by 2 or 3 or 5. Then call the method in your program and print out what it returns.
+
+def divisible
+    answer = []
+    (1...100).each do |number|
```

Comment:
The assignment doesn't make it clear whether or not you're supposed to include 100 in the answer (it's divisible by both 2 and 5), but FYI the `...` excludes the upper bound, so 1...100 goes from 1 to 99. For 1 to 100, you'd want:

```suggestion
    (1..100).each do |number|
```